src/App.scss

```diff
@@ -27,5 +31,81 @@ h1{
 }
 
 
+//Remove Presets
+
+/* Box sizing rules */
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+/* Remove default padding */
+ul[class],
+ol[class] {
+  padding: unset;
+}
+
+/* Remove default margin */
+html,
+a,
+h1,
+h2,
+h3,
+h4,
+p,
+ul[class],
+ol[class],
+figure,
+blockquote,
+dl,
+dd {
+  margin: unset;
+  font-size: unset;
+  text-decoration: unset;
+  font-weight: unset;
+  margin-block-start: unset;
+  margin-block-end: unset;
+  display: unset;
+  color: unset;
+  background-color: unset;
+}
+
+/* Set core body defaults */
+body {
+  min-height: 100vh;
+  scroll-behavior: smooth;
+  text-rendering: optimizeSpeed;
+  line-height: 1.5;
+}
+
+/* Remove list styles on ul, ol elements with a class attribute */
+ul[class],
+ol[class] {
+  list-style: none;
+}
+
+
+/* Make images easier to work with */
+img {
+  display: flex;
+  max-width: 100vw;
+  border-width: 0;
+  margin: 0;
+}
```

Comment:
I'd be a little suspicious about this.
Is there a reason to think that we'd want all of our images to behave this way by default?